purchases/build.gradle

```diff
@@ -4,6 +4,19 @@ apply plugin: 'org.jetbrains.dokka'
 
 android {
     namespace 'com.revenuecat.purchases.api'
+
+// Commenting out flavor temporarily after main -> bc5-support merge. We will need to fix integration tests
```

Comment:
Added a comment so we come back to fix these integration tests. They won't work after removing the latestDependencies/unityIAP flavors in the bc5-support branch.